programs/ssl/ssl_client2.c

```diff
@@ -1534,13 +1534,22 @@ int main( int argc, char *argv[] )
             {
                 sig_alg_list[i++] = MBEDTLS_TLS13_SIG_ECDSA_SECP521R1_SHA512;
             }
+#if defined(MBEDTLS_X509_RSASSA_PSS_SUPPORT)
```

Comment:
I do not see here guards for the other signature algorithms thus I'd suggest the same for RSASSA_PSS.